source/core/src/main/com/csse3200/game/inventory/items/AbstractItem.java

```diff
@@ -15,6 +17,8 @@ public abstract class AbstractItem {
     protected final int limit; // Must be non-negative
     protected int quantity; // Must be non-negative
 
+    protected Texture itemTexture;
```

Comment:
Just name this texture, and make it private (subclasses only need to be able to interact with it via a getter and setter.